docs/how-to/extract-packages.rst

```diff
@@ -0,0 +1,91 @@
+Extract packages
+================
+
+This article demonstrates how to extract the contents of debian packages.
+
+See also the article :doc:`/explanation/package-model` for a deeper 
+understanding of package formats.
+
+.. _ExtractSourcePackage:
+
+Extract a source package 
+------------------------
+
+This section demonstrates how to extract the content of a source package.
+
+.. note::
+
+    A source package archive has the file ending `.dsc`.
+    See also the man page :manpage:`dsc(5)` for further information.
+
+.. important::
+
+    Make sure that you have the `dpkg-dev` package installed. 
+    To install it run the following commands in a terminal:
+
+    .. code-block:: bash
```

Comment:
```suggestion
    .. code-block:: none
```

Reply:
Why did you change the syntax of this and other code blocks from `bash` to `none`?

Reply:
Because:
- It's not semantically correct to use (Bash) syntax highlighting on command-line examples. It's not a Bash code block (i.e. a piece of Bash code where the user could expect to see syntax highlighting IRL).
- The highlighting that results from having `bash` specified is not particularly helpful. E.g.:
   
   ![image](https://github.com/user-attachments/assets/d5b64f7a-8ff5-42c6-90c3-e5d61ceed0d3)

   (It's not that bad in this example, but elsewhere, with numbers involved, for example, it becomes more confusing.)

But the main reason is that it's not intuitive for the user: they do not see their commands syntax-highlighted as they type/paste them on the command line.